src/App.css

```diff
@@ -0,0 +1,192 @@
+/* stylelint-disable declaration-colon-newline-after */
+
+.NavBar {
+  background-color: #fff;
+  color: black;
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+}
+
+.coins-container {
+  margin: 0;
+  display: flex;
```

Comment:
- [ ] You have used `display:flex` 7 times falls under category of repeating yoursef, which is CSS bad practice, kindly create a class with display flex and use it whenever you need it